doc/rest_api.rst

```diff
@@ -6,12 +6,18 @@ To ease the integration with external sources (software or other),
 Modoboa provides a REST API.
 
 Every installed instance comes with a ready-to-use API and a
-documentation. You will find them using the following url patterns:
+documentation. 
 
-* API: *http://<hostname>/api/v1/*
-* Documentation: *http://<hostname>/docs/api/*
+**You will find them using the following url patterns:**
 
-An `example <https://demo.modoboa.org/docs/api/>`_ of this
+* New API+Docs: *https://<hostname>/api/schema-v2/swagger/*
```

Comment:
New API is also available under https://<hostname>/api/v2/

Reply:
> New API is also available under `https://<hostname>/api/v2/`

At the time of editing this it wasn't available under that route.